media-libs/libopenaptx/libopenaptx-0.2.0-r1.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
```

Comment:
You should probably do libfreeaptx too.

Reply:
Now with gstreamer PR 1667 merged I added a patch from that change as well and updated `gst-plugins-openaptx` dependency to include `libfreeaptx` alternative.